controllers/vrg_pvc_selector.go

```diff
@@ -14,29 +13,33 @@ type PvcSelector struct {
 	NamespaceNames []string
 }
 
-func pvcNamespaceNamesDefault(vrg ramen.VolumeReplicationGroup) []string {
-	return []string{vrg.Namespace}
-}
+// pvcNamespaceNamesDefault returns the default pvc namespaces for the VRG.
+// If the VRG namespace is the Ramen operands namespace, then the protected namespaces are used.
+// In the else cases, vrg in application namespace or the ramen operator namespace, the VRG namespace is used.
+func pvcNamespaceNamesDefault(vrg ramen.VolumeReplicationGroup, ramenConfig ramen.RamenConfig) []string {
```

Comment:
This is a slight silent ignore of `ProtectedNamespaces` if set on the VRG. E.g a user namespace VRG (not created by DRPC), lists a set of NS then this function would silently ignore it than error out.

I was thinking if we could validate early regarding this mismatch and report an error, it would make it cleaner to process mentally.